Performing C SOURCE FILE Test CMAKE_HAVE_LIBC_PTHREAD failed with the following output:
Change Dir: /home/berenice/Documents/txa-dvic-projects-tutos/build/CMakeFiles/CMakeTmp

Run Build Command(s):/usr/bin/make cmTC_969af/fast && /usr/bin/make  -f CMakeFiles/cmTC_969af.dir/build.make CMakeFiles/cmTC_969af.dir/build
make[1] : on entre dans le répertoire « /home/berenice/Documents/txa-dvic-projects-tutos/build/CMakeFiles/CMakeTmp »
Building C object CMakeFiles/cmTC_969af.dir/src.c.o
/usr/bin/cc   -DCMAKE_HAVE_LIBC_PTHREAD -o CMakeFiles/cmTC_969af.dir/src.c.o -c /home/berenice/Documents/txa-dvic-projects-tutos/build/CMakeFiles/CMakeTmp/src.c
Linking C executable cmTC_969af
/home/berenice/.local/lib/python2.7/site-packages/cmake/data/bin/cmake -E cmake_link_script CMakeFiles/cmTC_969af.dir/link.txt --verbose=1
/usr/bin/cc  -DCMAKE_HAVE_LIBC_PTHREAD -rdynamic CMakeFiles/cmTC_969af.dir/src.c.o -o cmTC_969af 
CMakeFiles/cmTC_969af.dir/src.c.o : Dans la fonction « main » :
src.c:(.text+0x3e) : référence indéfinie vers « pthread_create »
src.c:(.text+0x4a) : référence indéfinie vers « pthread_detach »
src.c:(.text+0x56) : référence indéfinie vers « pthread_cancel »
src.c:(.text+0x67) : référence indéfinie vers « pthread_join »
src.c:(.text+0x7b) : référence indéfinie vers « pthread_atfork »
collect2: error: ld returned 1 exit status
CMakeFiles/cmTC_969af.dir/build.make:105: recipe for target 'cmTC_969af' failed
make[1]: *** [cmTC_969af] Error 1
make[1] : on quitte le répertoire « /home/berenice/Documents/txa-dvic-projects-tutos/build/CMakeFiles/CMakeTmp »
Makefile:140: recipe for target 'cmTC_969af/fast' failed
make: *** [cmTC_969af/fast] Error 2


Source file was:
#include <pthread.h>

void* test_func(void* data)
{
  return data;
}

int main(void)
{
  pthread_t thread;
  pthread_create(&thread, NULL, test_func, NULL);
  pthread_detach(thread);
  pthread_cancel(thread);
  pthread_join(thread, NULL);
  pthread_atfork(NULL, NULL, NULL);
  pthread_exit(NULL);

  return 0;
}

Determining if the function pthread_create exists in the pthreads failed with the following output:
Change Dir: /home/berenice/Documents/txa-dvic-projects-tutos/build/CMakeFiles/CMakeTmp

Run Build Command(s):/usr/bin/make cmTC_4dfea/fast && /usr/bin/make  -f CMakeFiles/cmTC_4dfea.dir/build.make CMakeFiles/cmTC_4dfea.dir/build
make[1] : on entre dans le répertoire « /home/berenice/Documents/txa-dvic-projects-tutos/build/CMakeFiles/CMakeTmp »
Building C object CMakeFiles/cmTC_4dfea.dir/CheckFunctionExists.c.o
/usr/bin/cc   -DCHECK_FUNCTION_EXISTS=pthread_create -o CMakeFiles/cmTC_4dfea.dir/CheckFunctionExists.c.o -c /home/berenice/.local/lib/python2.7/site-packages/cmake/data/share/cmake-3.18/Modules/CheckFunctionExists.c
Linking C executable cmTC_4dfea
/home/berenice/.local/lib/python2.7/site-packages/cmake/data/bin/cmake -E cmake_link_script CMakeFiles/cmTC_4dfea.dir/link.txt --verbose=1
/usr/bin/cc  -DCHECK_FUNCTION_EXISTS=pthread_create -rdynamic CMakeFiles/cmTC_4dfea.dir/CheckFunctionExists.c.o -o cmTC_4dfea  -lpthreads 
/usr/bin/ld : ne peut trouver -lpthreads
collect2: error: ld returned 1 exit status
CMakeFiles/cmTC_4dfea.dir/build.make:105: recipe for target 'cmTC_4dfea' failed
make[1]: *** [cmTC_4dfea] Error 1
make[1] : on quitte le répertoire « /home/berenice/Documents/txa-dvic-projects-tutos/build/CMakeFiles/CMakeTmp »
Makefile:140: recipe for target 'cmTC_4dfea/fast' failed
make: *** [cmTC_4dfea/fast] Error 2



